supportedBackends/spark/src/main/scala/cromwell/backend/impl/spark/SparkJobExecutionActor.scala

```diff
@@ -0,0 +1,207 @@
+package cromwell.backend.impl.spark
+
+import java.nio.file.FileSystems
+import java.nio.file.attribute.PosixFilePermission
+
+import akka.actor.Props
+import cromwell.backend.BackendJobExecutionActor.{BackendJobExecutionResponse, FailedNonRetryableResponse, SucceededResponse}
+import cromwell.backend.impl.spark.SparkClusterProcess._
+import cromwell.backend.io.JobPaths
+import cromwell.backend.sfs.{SharedFileSystem, SharedFileSystemExpressionFunctions}
+import cromwell.backend.{BackendConfigurationDescriptor, BackendJobDescriptor, BackendJobExecutionActor}
+import cromwell.core.{TailedWriter, UntailedWriter}
+import wdl4s.parser.MemoryUnit
+import wdl4s.util.TryUtil
+
+import scala.concurrent.{Future, Promise}
+import scala.sys.process.ProcessLogger
+import scala.util.{Failure, Success, Try}
+import scala.language.postfixOps
+
+object SparkJobExecutionActor {
+  val fileSystems = List(FileSystems.getDefault)
```

Comment:
You could also explicitly refer to the right, e.g `file.FileSystems` and change the imports